merkle/simple_tree.go

```diff
@@ -22,6 +22,7 @@ For larger datasets, use IAVLTree.
 
 */
 
+// nolint: goimports
```

Comment:
same comment

Reply:
file not imported anywehre

Reply:
I don't understand what that means. its a pkg for external use.

Reply:
so leave the nolint and move on?

Reply:
Are you suggesting I allow lines of code in my codebase that I don't understand?